src/validation.h

```diff
@@ -560,7 +560,7 @@ class Chainstate
     //! Cached result of LookupBlockIndex(*m_from_snapshot_blockhash)
     mutable const CBlockIndex* m_cached_snapshot_base GUARDED_BY(::cs_main){nullptr};
 
-    std::atomic_bool m_prev_script_checks_logged{true};
+    std::optional<bool> m_prev_script_checks_logged{};
```

Comment:
https://github.com/bitcoin/bitcoin/pull/33336/commits/64ba2821b5a55aa6d51fbb7a16470d4f7c41c92e:
I believe you now need to add `GUARDED_BY(::cs_main)` here to use Clang's Thread Safety Analysis

Reply:
Excellent, added you as a coathor, thanks.